### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by daudi13**
#In this project we:

- Added a contact form
- Used the correct Github Flow
- Used Linters to check for semantic errors
- used the Figma design provided

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -176,6 +176,22 @@ <h2 class="headline-section__sub-title">
...
+				<div class="outro__box">
+					<h2 class="healine-section__title white">Contact me</h2>
+					<p class="headline-section__desc white">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+				</div>
+				<form action="https://formspree.io/f/" method="post">
+                    <label for="text"><input  type="text" required placeholder="Full Name" maxlength="30" id="text"></label>
+                    <label for="email"><input name="Email" id="email" type="email" required maxlength="30" placeholder="Your email"></label>
+					<textarea cols="30" rows="10" maxlength="500" placeholder="Write your message here"></textarea>
+                    <div class="btn__submit">
+						<button type="submit">Get in touch</button>
```
Comment (brytebee) [inline index.html]: - [ ] **Required:** Please verify that the form is submitted properly. When I tried to submit it, I got the error below.
image

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @daudi13 👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- proper form validations✔️
- amazing UI design and use of template✔️
- clean code✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] **Required:** Consider adding a live demo link.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (daudi13) [reply index.html]: I have just updated the form action

**Review by ReshyResh — APPROVED** [review]
# Hi team, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
#### Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**